resources/aws/elasticip.go

```diff
@@ -0,0 +1,45 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+const (
+	allocationID = "eipalloc-cc1657a5"
+	elasticIP    = "35.158.16.27"
```

Comment:
I assume this is for testing purposes and only because we are in an early stage.